modules/op-running-pipeline-and-task-run-pods-with-privileged-security-context.adoc

```diff
@@ -0,0 +1,69 @@
+[id='op-running-pipeline-and-task-run-pods-with-privileged-security-context']
+= Running pipeline run and task run pods with privileged security context
+:context: op-running-pipeline-and-task-run-pods-with-privileged-security-context
+
+toc::[]
+
+.Procedure
+
+To run a pod (resulting from pipeline run or task run) with `privileged` security context, do the following modifications:
+
+* Configure the associated user account or service account to have an explicit SCC. You can perform the configuration using any of the following methods:
+** Use the following OpenShift command:
++
+[source, terminal]
+----
+$ oc adm policy add-scc-to-user <scc-name> -z <service-account-name>
+----
+** Manually modifying the YAML for `RoleBinding` and `Role` or `ClusterRole`:
```

Comment:
We can drop the word 'Manually' instead, maybe add - 'Alternatively,' to make it clear that the following method is another way to do the same thing.
s/modifying/modify
Nit - suggest: the `RoleBinding` and `Role` or `ClusterRole` YAML files.

Reply:
Used "Alternatively".
I did not push the "YAML files" to the end. Instead, made some subtle changes that should make the meaning more clear/explicit.